simulations/agents/uniswap_agent.py

```diff
@@ -511,19 +805,51 @@ def __init__(
             token_a_address=token_a_address,
             token_b_address=token_b_address,
             mu=mu,
-            sigma=sigma,
+            sigma=0,
```

Comment:
Did you want to set this?

Reply:
Yes, I'm making the exploration deterministic now, so no sigma.